iocore/net/SSLUtils.cc

```diff
@@ -1953,8 +1953,9 @@ SSLMultiCertConfigLoader::load(SSLCertLookup *lookup)
   if (ec) {
     switch (ec.value()) {
     case ENOENT:
+      // missing config file is an acceptable runtime state
```

Comment:
We don't what to fail is ssl_multicert.config is unavailable?

Reply:
autest fails all over the place without allowing this. I think we generally try to strive to allow for missing config files (at least since I've been involved in the project)

Reply:
A server may not have any https remaps ... can imagine that requiring a blank ssl_multicert would be confusing.

Reply:
Ok.  Good point.